### PR TITLE
[WIP] Removed eco gem and installed ejs

### PR DESCRIPTION
**Pull request by rklymenko**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -49,7 +49,7 @@ gem 'bootstrap-sass', '~> 3.2.0.2'
 gem 'font-awesome-sass'
 gem 'bourbon'
 gem 'momentjs-rails'
-gem 'eco'
+gem 'ejs'
```
Comment (yivo) [inline Gemfile]: @rklymenko You should always lock gem to version range.

**Review by yivo — CHANGES_REQUESTED** [review]

**Comment by yivo** [comment]
@rklymenko Please fix specs.

**Comment by rklymenko** [comment]
@yivo I locked gem version. But for specs, I think need to create another branch, because issues not for this task, these issues exists in master branch

File: Gemfile [hunk 2]
```diff
@@ -49,7 +49,7 @@ gem 'bootstrap-sass', '~> 3.2.0.2'
 gem 'font-awesome-sass'
 gem 'bourbon'
 gem 'momentjs-rails'
-gem 'eco'
+gem 'ejs', '1.1.1'
```
Comment (yivo) [inline Gemfile:52]: @rklymenko I am sure you are not doing wright. Have you previously used RubyGems & Bundler?

Comment (yivo) [reply Gemfile:52]: Are you familiar with semver?

**Review by yivo — CHANGES_REQUESTED** [review]

**Comment by yivo** [comment]
@rklymenko You may check master branch or devel for working specs. Navigate to TravisCI and you will see all specs are passed.

**Comment by yivo** [comment]
@rklymenko You should also update the PR to target to correct branch as in specs.

File: Gemfile (hunk 2 above)
Comment (rklymenko) [reply Gemfile:52]: yes, I used RubyGems and Bundler but don't used semver

**Comment by rklymenko** [comment]
I checked specs in the master branch, and have 5 issues like in my branch

File: Gemfile (hunk 2 above)
Comment (yivo) [reply Gemfile:52]: @rklymenko You couldn't use Bundler without using semver.

**Comment by rklymenko** [comment]
`Failed examples:

rspec ./spec/features/market_spec.rb:58 # show account info user can fill order form by clicking on an existing orders in the order book
rspec ./spec/features/market_spec.rb:24 # show account info user can place a buy order by filling in the order form
rspec ./spec/features/market_spec.rb:41 # show account info user can place a sell order by filling in the order form
rspec ./spec/features/market_spec.rb:81 # show account info user can view his account balance
rspec ./spec/features/market_trade_history_spec.rb:14 # show account info user can cancel his own order

Randomized with seed 59637

slan@asus /var/www/peatio $ git status
On branch master
Your branch is up-to-date with 'origin/master'.
`

**Comment by yivo** [comment]
@rklymenko I don't see any issues with specs on master right now.

**Comment by yivo** [comment]
@rklymenko Any progress on it?

**Comment by yivo** [comment]
@rklymenko Any updates?

**Comment by rklymenko** [comment]
I am working on specs. I have issue only with selenium

**Comment by rklymenko** [comment]
Hi @yivo , I checked specs one more time. It's not my issues. They exist in master branch on my fork. Check please. And if you check one of specs - you see that issue different from my task.

**Comment by rklymenko** [comment]
@yivo And if you want I can work with it but in another issue and task

**Comment by yivo** [comment]
@rklymenko Hello. We don't have any broken specs now in master and in devel. I believe you should check the issue once again and update PR to match requirements.

**Pull request closed without merging**